desktop/src/main/java/bisq/desktop/main/overlays/windows/OfferDetailsWindow.java

```diff
@@ -253,21 +253,20 @@ private void addContent() {
             addConfirmationLabelLabel(gridPane, ++rowIndex, Res.get("offerDetailsWindow.myTradingAccount"), myPaymentAccount.getAccountName());
         } else {
             final String method = Res.get(paymentMethod.getId());
-            String methodWithBankId = method + bankId;
             String paymentMethodLabel = Res.get("shared.paymentMethod");
             if (countryCode != null && (isNationalBanks || isSpecificBanks || isSepa)) {
                 if (BankUtil.isBankIdRequired(countryCode))
                     addConfirmationLabelLabel(gridPane, ++rowIndex,
-                            paymentMethodLabel + " " + Res.get("offerDetailsWindow.offererBankId"),
```

Comment:
If you remove a translations please also remove it from `displayStrings.properties`. Otherwise we'll have more and more unnecessary strings to be translated. But in this case you probably need them anyways for the specific banks payment method